tests/monitors/expvar/expvar_test.py

```diff
@@ -12,26 +12,26 @@
 METADATA = Metadata.from_package("expvar")
 
 
-def test_expvar_included(expvar):
+def test_expvar_included(expvar_container):
```

Comment:
This is the IP of the container right?  If so, this needs to be called something along those lines because most of the existing tests use the name `*_cont[ainer]` to refer to the docker-py container instance, which is going to be really confusing.